runtimes/server-interface/logging.ts

```diff
@@ -1,7 +1,13 @@
+import { LogLevel } from '../runtimes/util/loggingUtil'
+
 /**
  * The logging feature interface
  */
 export type Logging = {
-    // TODO: Implement log levels
+    level: LogLevel
+    error: (message: string) => void
+    warn: (message: string) => void
+    info: (message: string) => void
     log: (message: string) => void
+    debug: (message: string) => void
```

Comment:
nit: we could add something as `...args: any` for convernience if we are already improving this interface

Reply:
I could add but logger which we are using lspConnection.console.log/error/warn/info/debug does not support passing extra arguments, they accept a single string argument only.